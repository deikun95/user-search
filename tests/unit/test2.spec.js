// import { shallowMount } from "@vue/test-utils";
// import UserItem from "@/components/UserItem";
// import UserDialog from "@/components/UserDialog";

// describe("UserDialog.vue", () => {
//   it("change open value after close button is pressed", () => {
//     const wrapper = shallowMount(UserDialog, {
//         propsData: {
            
//         }
//     });
//     // const wrapper2 = shallowMount(UserItem);
//     expect(wrapper.vm.open).toBe(true);
//     wrapper.find(".close-button").trigger("click");
//     expect(wrapper.vm.open).toBe(false);
//   });
// });
