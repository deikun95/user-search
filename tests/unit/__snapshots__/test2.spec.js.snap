// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserSearch renders the component 1`] = `
<div class="wrapper"><input type="text" placeholder="Please enter login..." class="search">
  <el-select placeholder="Select">
    <el-option label="Descending" value="down"></el-option>
    <el-option label="Ascending" value="up"></el-option>
  </el-select>
</div>
`;
